onvm/go.sh

```diff
@@ -48,13 +48,32 @@ fi
 
 shift 3
 
+# Verify that bc is installed
+if [[ $(dpkg-query -W -f='${Status}' bc 2>/dev/null | grep -c "ok installed") == 0 ]]
+then
+    echo "Error: bc is not installed. Install using:"
+    echo "  sudo apt-get install bc"
+    echo "See dependencies for more information"
+    exit 1
+fi
+
 if [ -z "$nf_cores" ]
 then
     usage
 fi
 
+ports_bin=$(echo "obase=2; ibase=16; $ports" | bc)
```

Comment:
Could you just use `obase=10` to go directly to decimal?

Reply:
I could be interpreting your comment wrong, but I'm not sure if that's exactly what we want. Based on what I found when testing this, the hexadecimal port mask should be translated to binary, and we determine which ports are desired by checking if the digit is "1". For example:
```sh
./go.sh 0,1,2 3 0xF8 -s stdout
```
would enable ports 0 and 1 because: `3 -> 11 -> digit 1, digit 0`. Accordingly:
```sh
./go.sh 0,1,2 4 0xF8 -s stdout
```
would enable only port 2 because: `4 -> 100 -> digit 2, not digit 1, not digit 0`. Since I convert the port mask to binary, we get the digits in the binary format so all I have to do is determine if each digit is a "1", and update the count of requested ports based on that result. I do this check by performing simple modulo arithmetic by treating the binary number like it's in decimal form, which allows me to check if the last number is 1. Accordingly, if I use `obase=10` to convert directly to decimal, the examples I used of `3` and `4` would still be `3` and `4`, respectively, which doesn't tell me much about the ports requested in the binary format. Let me know what you think.

Reply:
Yep you are right. I wasn’t thinking straight. 👍